packages/cli/src/handlers/preview.ts

```diff
@@ -290,13 +302,15 @@ export const startPreviewHandler = async (
 
         // Create
         console.error(`Creating new project preview ${projectName}`);
-        const project = await createProject({
+        const results = await createProject({
             ...options,
             name: projectName,
             type: ProjectType.PREVIEW,
             copiedFromProjectUuid: config.context?.project,
         });
 
+        const project = results?.project;
```

Comment:
We need the same warning in here. 
Users can create projects with both `lightdash preview` and `lightdash start-preview`. The main difference is that the first one keeps checking if files change to redeploy the project.